Symfony/CS/Tests/Fixtures/Integration/priority/phpdoc_no_package,phpdoc_regular_header.test

```diff
@@ -0,0 +1,23 @@
+--TEST--
+Integration of fixers: phpdoc_regular_header,header_comment.
+--CONFIG--
+level=none
+fixers=phpdoc_no_package,phpdoc_regular_header
+--EXPECT--
+<?php
+/*
+ * Docblock without phpdoc
+ *
```

Comment:
That is wrong IMO. It should be converted straight to a comment first, then not modified.